mods/ra/rules/defaults.yaml

```diff
@@ -249,8 +249,14 @@
 	Selectable:
 		Bounds: 24, 24
 	Targetable:
-		TargetTypes: Ground, Repair, Vehicle
+		TargetTypes: Ground, Vehicle
 		RequiresCondition: !parachute
+	Targetable@REPAIR:
+		TargetTypes: REPAIR
+		RequiresCondition: !parachute && damaged
```

Comment:
Style nit: Please move all these `RequiresCondition`s to the top of their definitions.